src/main/webapp/WEB-INF/jsp/help.jsp

```diff
@@ -0,0 +1,181 @@
+    <script src="${context}/js/scripts.js"></script>
+    <title>Help</title>
+</head>
+<body>
+<nav class="navbar navbar-default">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="${context}/roster/${selectedSectionId}">K-State Attendance</a>
+        </div>
+        <ul class="nav navbar-nav">
+            <li><a id="classSetupLink" href="${context}/courseConfiguration/${selectedSectionId}">Setup</a></li>
+            <li><a id="attendanceSummaryLink" href="${context}/attendanceSummary/${selectedSectionId}">Attendance
+                Summary</a></li>
+            <li><a id="rosterLink" href="${context}/roster/${selectedSectionId}">Class Roster</a></li>
+            <li class="active"><a id="helpLink" href="${context}/help/${selectedSectionId}">Help</a></li>
+        </ul>
+    </div>
+</nav>
+
+<h1>Start Up Guide for K-State Attendance</h1>
+<hr>
+
+<div class="panel-group form-div" id="accordion" role="tablist" aria-multiselectable="true">
+    <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="headingOne">
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne"
+                   aria-expanded="false" aria-controls="collapseOne">
+                Saving a Setup
+                </a>
+            </h4>
+        </div>
+        <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+            <div class="panel-body">
+                <img src="${context}/img/Setup.png" border="5">
+                <hr>
+                <p>Before you can push a section's attendance to the Canvas gradebook, a Setup must be associated with
+                the section. The Setup consists of the three fields that open up when the "Convert Attendance to
+                Assignment" box is checked: the <em>Assignment Name </em>, the <em>Total Points</em>, and the
+                <em>Attendance Weights</em> (pictured above).</p>
+                <p><strong>NOTE:</strong> If you change any of the required fields on the Setup page, you will need to
+                save the setup again before pushing to the Canvas Gradebook.</p>
+            </div>
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="headingTwo">
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo"
+                   aria-expanded="false" aria-controls="collapseTwo">
+                    Pushing Grades
+                </a>
+            </h4>
+        </div>
+        <div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
+            <div class="panel-body">
+                <img src="${context}/img/PushModal.png" border="5">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "border" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)